packages/@unimodules/core/src/__tests__/deprecate-test.ts

```diff
@@ -1,4 +1,4 @@
-import * as Errors from '../Errors';
+import { CodedError, deprecate } from '../index';
```

Comment:
Probably we could but when I was looking for a solution to 
```
error TS5055: Cannot write file '/Users/tomasz/Work/expo/packages/@unimodules/react-native-adapter/build/EventEmitter.d.ts' because it would overwrite input file.
```
I've seen some comments this can happen also when importing a package itself.
Anyway, I'll try it out.